tasks/npm.yml

```diff
@@ -19,9 +19,10 @@
     - name: Prepare node image with pax flags
       docker_image:
         name: "node_grsec:{{ django_stack_node_ver }}"
-        dockerfile: "{{ role_path }}/docker/NodeDockerfile"
+        dockerfile: "{{ django_stack_node_dockerfile }}"
```

Comment:
We discussed this in person... regarding the trade-off of trying to do this via templating versus specifying a new dockerfile. Agreed that this approach is path of least resistance for now though and the right move to keep moving forward 👍